dist/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>TVmaze</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+<script defer src="main.js"></script></head>
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <div class="logo">
+        <a href="">
+          <img src="" alt="logo">
+        </a>
+      </div>
+      <div class="menu-links">
+        <a href="" class="menu-link" >
+          Movies<span id="shows-count"></span>
+        </a>
+        <a href="" class="menu-link">Schedule</a>
+      </div>
```

Comment:
- [ ] You did some great work here 👏 . However, lets make it more user friendly in other to have a better acceptance by  spacing the navigation headers and aligning them to be on part with the design given.

## Your view:
![image](https://user-images.githubusercontent.com/50990925/233756519-e77116da-65ab-4c27-aa68-6b56f2fab411.png)

## Wireframe:
![image](https://user-images.githubusercontent.com/50990925/233756532-3e248a81-1f87-498c-adae-7a9d4ae463b4.png)